nerdtree_plugin/fs_menu.vim

```diff
@@ -203,7 +203,7 @@ endfunction
 function! NERDTreeListNode()
     let treenode = g:NERDTreeFileNode.GetSelected()
     if treenode != {}
-        let metadata = split(system('ls -ld ' . shellescape(treenode.path.str())), '\n')
+        let metadata = split(system('ls -ldh ' . shellescape(treenode.path.str())), '\n')
```

Comment:
Yeah, I suspected that would happen. I was trying to develop a better regex. It works with grep, but not sed. Really strange. Assuming it can be fixed to work with sed, it will also comma-fy the years, so this ultimately isn't a solution either.

```
ls -l | grep '\(^\|\s\)\(\d\+\)\(\d\{3\}\)\(\(,\d\{3\}\)*\)\(\s\|$\)'
ls -l | sed -e :a -e 's/\(^\|\s\)\(\d\+\)\(\d\{3\}\)\(\(,\d\{3\}\)*\)\(\s\|$\)/\1\2,\3\4\6/;ta'
```